tests/pool_test.py

```diff
@@ -538,7 +538,7 @@ class MyPool(ConnectionsPool):
         server.tcp_address,
         minsize=10, loop=loop)
 
-    with await pool as conn:
+    with (await pool) as conn:
```

Comment:
Why not `async with`?

Reply:
It could be rewritten to `async with pool.get() as conn`. The deprecation warning is not triggered on `ConnectionsPool` objects. Rewriting as `async with pool as conn` generates an `AttributeError` because `__aexit__` is not implemented like in `_AsyncConnectionContextManager`.

Reply:
Well, then it's a part of the fix.